src/protocol.c

```diff
@@ -91,6 +91,77 @@ check_host_origin(struct lws *wsi) {
     return false;
 }
 
+/* Parse the request URI to retrieve client-specified URI parameters.
+The URI should be in the format: /path?command=<command>&arg1=<arg>&argN=<arg>
+*/
+int
+parse_client_command(struct lws *wsi, struct tty_client *client) {
```

Comment:
Yes, I originally considered sending the command and arguments as json in the websocket connection. I'll try and make this change.